db_test.go

```diff
@@ -425,6 +426,52 @@ func (s *SurrealDBTestSuite) TestSmartMarshalQuery() {
 	})
 }
 
+func (s *SurrealDBTestSuite) TestConcurrent() {
+	var wg sync.WaitGroup
+	totalcoroutine := 100
```

Comment:
this can be totalGoroutines/totalThreads (camel case, coroutines are more towards the C#/Kotlin terminology) or totalConcurrentRequests (more descriptive name for the variable).

Reply:
> camel case

Thanks for the review I missed that camel case, I fix it.

> "totalThreads

I don't think this is the best way to go with cuz goroutine are not managed by kernel-like threads.
I go with "totalGoroutines".